### DISCUSSION TITLE
Selecting and editing inner component

### DISCUSSION THREAD

**Discussion by madhuvinod23** [jgraph/drawio#2097] [Q&A] [answered]
Hello - If i am creating a architecture diagram, is there a way to select and edit a inner box/component? When i click on the inner component outer component gets selected. If i had to edit the inner component, i need to drag the component out and edit it, drag it back to previous position as of now. I am using a draw.io app on a mac. Thank you very much in advance.

**Comment by madhuvinod23** [comment, marked as answer]
Found a way to do what I was trying. When you click on outer box, you will see 3 tabs on your right named "style", "text" and "arrange" to modify the box. click on arrange tab and then select "To Back". This enables us to select any inner component/box to edit.

**Reply by hi2u** [reply]
Have you come across any workaround for when you want to select multiple inner items (via dragging a selection rectangle).
